ArcGISRuntimeSDKQt_CppSamples/Layers/AddDynamicEntityLayer/AddDynamicEntityLayer.h

```diff
@@ -17,6 +17,7 @@
 #ifndef ADDDYNAMICENTITYLAYER_H
 #define ADDDYNAMICENTITYLAYER_H
 
+#include "QtGui/qevent.h"
```

Comment:
This shouldn't be needed. Forward declare QMouseEvent instead.

Reply:
removed.